apache/superset
# [WIP] feat(explore): Implement using multiple time column filters in charts

### SUMMARY
Implements new interface for time section in Explore. Now user can use multiple time columns for filtering purposes, set one of them as X-Axis and select different time grains for each time column.

TODO: hide x-axis checkbox for non-timeseries charts, fix tests, implement backend, write a migration.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:
![image](https://user-images.githubusercontent.com/15073128/149919133-a523d9e4-c562-4d84-948a-968baebc13db.png)

After:
![image](https://user-images.githubusercontent.com/15073128/149918697-6538c299-27ee-4618-974d-867b7b4bcff3.png)


### TESTING INSTRUCTIONS
1. Make sure that Time filter in native filters works like before
2. Make sure that time range pre-filter in native filters config modal works as before
3. In order to properly test the feature in Explore we need to wait for backend implementation

### ADDITIONAL INFORMATION
